README.md

```diff
@@ -1,2 +1,16 @@
 # microverse-portfolio1
-This repo was created for my portfolio
+# My portfolio
+This project was created for the creation of my portfolio.
+
+# LANGUAGES USED
+1. HTML
+2. CSS
+3. Linters(Lighthouse, Webhint, Stylelint
+# PREREQUISITES
+1. Prior knowledge of HTML and CSS.
+2. Setup: clone direakanbi/microverse-portfolio1 into your local machine
+3. Run index.html on your local server.
+
+# Live Demo Link.
+http://127.0.0.1:5500/portfolio1/index.html#
```

Comment:
- [ ] I suggest using the Microverse README template to create a more robust README 🙂